ansible/roles/ocp4-workload-cost-uploader/tasks/workload.yml

```diff
@@ -74,17 +74,17 @@
 
 - name: Unzip master.zip
   unarchive:
-    remote_src: yes
+    remote_src: true
     src: "/home/ec2-user/korekuta-master.zip"
     dest: "/home/ec2-user/"
     creates: "/home/ec2-user/korekuta-master/"
 
 - name: Run Korekuta Setup for this cluster
   command: |
```

Comment:
This is kind of weird and surprising that this works! Normally line breaks in a command would be a problem and the `>-` would be required, but I guess that is not how the `command` module works... really though, this should not have line breaks in the command. Let's switch this.